distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -1,17 +1,11 @@
-import project.Data.Pair.Pair
```

Comment:
- clearly there is a lot of _unused imports_
- they can be removed without any functional impact
- reducing the transitive closure without any cost being payed
- some tooling is needed 
    - clearly regex isn't enough
    - related: #6520 
    - related: #5493 
    - but we don't seem to report unused imports in general